hikari/commands.py

```diff
@@ -136,6 +137,22 @@ class CommandOption:
         lowercase.
```

Comment:
What do you mean by this? Should I add `This will match the regex ^[\w-]{1,32}$ in Unicode mode and will be lowercase.` to the docstring of `name_localizations` as well?

Reply:
that's not the current regex; this command localisation feature changed the regex to `^[-_\p{L}\p{N}\p{sc=Deva}\p{sc=Thai}]{1,32}$`.

Reply:
honestly, I feel like there's no reason to explain the limitations with regex anymore.